bottom-bar/src/main/java/com/roughike/bottombar/BottomBarBadge.java

```diff
@@ -203,8 +203,11 @@ private void adjustPositionAndSize(View tabToAddTo) {
         setTranslationY(10);
 
         int size = Math.max(getWidth(), getHeight());
-        getLayoutParams().width = size;
-        getLayoutParams().height = size;
+
+        ViewGroup.LayoutParams params = getLayoutParams();
+        params.width = size;
+        params.height = size;
+        setLayoutParams(params);
```

Comment:
This call is creating an infinite loop because at above line 188 we have a global layout listener which calls this method (`adjustPositionAndSize`).

Please revert (since it's causing issues in the activity/fragment lifecycles) and figure out another way to fix.